docs/aws/guides/least-privileged-provisioning-role.md

```diff
@@ -0,0 +1,55 @@
+# Least Privileged Provisioning Role
```

Comment:
`README.md` under `/aws/guides` is empty... so GitBook will render a default page; so, no explicit link to this new section? is that OK